l10n_de_toponyms/__manifest__.py

```diff
@@ -16,6 +16,7 @@
     "license": "AGPL-3",
     "data": [
         "wizard/l10n_de_toponyms_wizard.xml",
+        "wizard/l10n_de_toponyms_zipcodes.xml",
```

Comment:
As far as I know is this just an example file you can get using the `tools/gen_toponyms_geonames.py`. This shouldn't be included in the data. Furthermore this are changes in another module and shouldn't be in this PR to keep the PR clean.

Reply:
In fact i stumbled over it as i comitted l10n_de_tax_statement as error notifications. Obviously these issues didn't happen as the original authors committed that module.  Actually i don't know if errors belonging to that module persists.